examples/tfengine/gke_cluster.hcl

```diff
@@ -0,0 +1,141 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# {{$recipes := "../../templates/tfengine/recipes"}}
+
+data = {
+  parent_type      = "organization" # One of `organization` or `folder`.
+  parent_id        = "12345678"
+  billing_account  = "000-000-000"
+  state_bucket     = "example-terraform-state"
+  storage_location = "us-central1"
+  compute_region   = "us-central1"
+}
+
+template "networks" {
+  recipe_path = "{{$recipes}}/project.hcl"
+  output_path = "./gke_cluster/networks"
+  data = {
+    project = {
+      project_id         = "example-networks"
+      is_shared_vpc_host = true
+    }
+    resources = {
+      compute_networks = [{
+        name = "network"
+        subnets = [
+          {
+            name     = "gke-subnet"
+            ip_range = "10.2.0.0/16"
+            secondary_ranges = [
+              {
+                name     = "pods-range"
+                ip_range = "172.16.0.0/14"
+              },
+              {
+                name     = "services-range"
+                ip_range = "172.20.0.0/14"
+              }
+            ]
+          }
+        ]
+      }]
+    }
+  }
+}
+
+template "cluster" {
+  recipe_path = "{{$recipes}}/project.hcl"
+  output_path = "./gke_cluster/cluster"
+  data = {
+    project = {
+      project_id = "example-apps"
+      shared_vpc_attachment = {
+        host_project_id = "example-networks"
+        subnets = [{
+          name = "gke-subnet"
+        }]
+      }
+    }
+    resources = {
+      gke_clusters = [{
+        name                   = "gke-cluster"
```

Comment:
name = "example-cluster"

Reply:
I know it's named gke-cluster in the team.hcl, but example-cluster seems more consistent with other resources you created here, such as example-sa

Reply:
Done.